pikelet/src/surface/projections/pretty.rs

```diff
@@ -49,7 +49,10 @@ where
         Term::Let(_, bindings, term) => (alloc.nil())
             .append("let")
             .append(alloc.intersperse(
-                bindings.iter().map(|t| pretty_term_prec(alloc, &t.2, prec)),
+                bindings.iter().map(|t| match t {
+                    Item::Declaration((_, _, r#type)) => pretty_term_prec(alloc, &r#type, prec),
+                    Item::Definition((_, _, term)) => pretty_term_prec(alloc, &term, prec),
```

Comment:
I think this can perhaps for now get away with, not sure.
```
Item::Declaration((_, _, term)) | Item::Definition((_, _, term) => pretty_term_prec(alloc, &term, prec),
```